src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -147,14 +147,16 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
             </FormControl>
           </Grid>
         </Grid>
-        <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
-            Create
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
-            Cancel
-          </Button>
-        </Box>
+        <Grid container justifyContent="flex-end">
+          <Box display="flex" sx={{ mt: 2 }}>
```

Comment:
you don't need the grid, just put the justifyContent="flex-end" on the Box component, and it should do the same thing